packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Base from 'terra-base';
-import Field from '../../../Field';
+import Field from '../../../../lib/Field';
```

Comment:
How come we are using the lib folder? It looks like the other files are using the source file.

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-f908e7da845ac45c958b8a06974c63c9R1).